packages/cdk-routing-split/test/steps/then.ts

```diff
@@ -20,3 +20,23 @@ export const item_exists_in_CdkRoutingSplitTable = async (expectedItem: any) =>
     expect(response.Items.length).toEqual(1);
     expect(response.Items[0]).toStrictEqual(expectedItem);
 };
+
+export const item_not_in_CdkRoutingSplitTable = async (expectedItem: any) => {
```

Comment:
Rename this function to item_does_not_exist_in_CdkRoutingSplitTable, to be paired with name of function above.